addon/components/bread-crumb.js

```diff
@@ -14,6 +14,7 @@ export default Component.extend({
   layout,
   tagName: 'li',
   classNameBindings: ['crumbClass'],
+  i18n: false,
```

Comment:
Is there a conventional way of doing this yet? Maybe @jasonmit ([ember-intl](https://github.com/jasonmit/ember-intl) author) has some advice for things like this?